### PR TITLE
feat: add build CI for windows

### PR DESCRIPTION
**Pull request by yihong0618**
running_page use tzfpy now, and we found it does not support windows.
image

pprof can not compile on windows.
1. need https://github.com/ringsaturn/tzf-rs/pull/36 merge first
2. windows test can not pass for now. because we also need citiespy support windows too then we can add windows test ci

### CODE REVIEW

**Comment by yihong0618** [comment]
Passed CI
https://github.com/yihong0618/tzfpy/actions/runs/4003349630/jobs/6871371815

**Comment by ringsaturn** [comment]
Thanks, I will check citiespy’s Windows build.

**Comment by yihong0618** [comment]
> Thanks, I will check citiespy’s Windows build.

when approve I will change the version in `toml`

**Comment by ringsaturn** [comment]
@yihong0618 Please bump tzf-rs to [v0.2.1](https://github.com/ringsaturn/tzf-rs/releases/tag/v0.2.1)

**Comment by yihong0618** [comment]
test added waiting for `citiespy`

**Comment by ringsaturn** [comment]
citiespy Windows support is ready: [0.3.3](https://pypi.org/project/citiespy/0.3.3/)

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -52,7 +54,7 @@ jobs:
       - uses: messense/maturin-action@v1
         with:
           command: build
-          args: --release -o dist --universal2 --find-interpreter
```
Comment (ringsaturn) [inline .github/workflows/CI.yml:55]: Why remove `universal2`?

Comment (yihong0618) [reply .github/workflows/CI.yml:55]: for test will try to bring it back

File: .github/workflows/Test.yml [hunk 2]
```diff
@@ -39,10 +40,23 @@ jobs:
...
           pip install -r requirements_dev.txt
           maturin develop
           pytest --cov
+
+      - name: Install tzfpy and test(windows)
+        shell: bash
+        # for windows need maturin build
+        if: matrix.os == 'windows-latest'
+        run: |
+          . venv/Scripts/activate.bat
```
Comment (ringsaturn) [inline .github/workflows/Test.yml:57]: venv is not required if use `maturin build`

Comment (yihong0618) [reply .github/workflows/Test.yml]: just found that....

**Comment by yihong0618** [comment]
finally passed in my actions...CI is hard work...

**Review by ringsaturn — APPROVED** [review]
Thanks for fighting with CI!

**Comment by ringsaturn** [comment]
It’s ready!

https://pypi.org/project/tzfpy/0.12.1/

**Pull request merged**